package.json

```diff
@@ -18,10 +18,12 @@
     "prepare": "webpack --config webpack.config.js --mode=production",
     "build:production": "webpack --config webpack.config.js --mode=production",
     "build:development": "webpack --config webpack.config.js --mode=development",
-    "lint": "eslint ./src/"
+    "lint": "eslint ./src/",
+    "test": "jest"
```

Comment:
might want to add a `pretest` script as well that runs the lint script. might as well catch those errors early :)

also, if eslint takes a while you can also add ` --fix --cache` to the end of the lint script above

Reply:
Went ahead and added the pretest script :+1: